src/index.js

```
@@ -0,0 +1,75 @@
+require('dotenv').config();
+const express = require('express');
+const mongoose = require('mongoose');
+const cors = require('cors');
+const morgan = require('morgan');
+const routes = require('./routes');
+const swaggerDocs = require('./config/swagger');
+
+const app = express();
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)